core/src/main/java/org/elasticsearch/action/admin/indices/analyze/TransportAnalyzeAction.java

```diff
@@ -189,6 +190,44 @@ public static AnalyzeResponse analyze(AnalyzeRequest request, String field, Anal
 
             analyzer = new CustomAnalyzer(tokenizerFactory, charFilterFactories, tokenFilterFactories);
             closeAnalyzer = true;
+        } else if (request.normalizer() != null ||
+            ((request.tokenFilters() != null && request.tokenFilters().size() > 0)
+                || (request.charFilters() != null && request.charFilters().size() > 0))) {
+            // normalizer + (tokenizer/analyzer) = no error, just ignore normalizer param
```

Comment:
I will add check logic in request.validate() method.
Unfortunately, it is not always called. If you call shardOperation yourself directly, validate() method is not called.